tools/packages.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Cake" version="0.19.4" />
+  <package id="Cake" version="0.25.0" />
```

Comment:
shouldn't this be `0.28.1` to match `Cake.Openshift.csproj`?

Reply:
The version of Cake that is used to build the addin, doesn't need to be the same as the one that the addin is compiled against.  In a likelihood, there will always be a slight lag, since Cake.Recipe needs to wait for all addins to support a new version of Cake, before itself can update to it.

In this instance, there were breaking changes in 0.28.0 of Cake that directly effect Cake.Recipe.

Reply:
I thought it might be something like that
just wanted to be sure 👍

Reply:
@Meir017 Unfortunately that will break the build at the moment as Cake.Recipe only supports 0.25.0 until [cake-contrib/Cake.Recipe#207](https://github.com/cake-contrib/Cake.Recipe/issues/207) gets fixed.